package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@vrbo/service-client",
   "description": "A general purpose http client built with extensibility in mind. It also features lifecycle hooks, dynamic hostname resolution, and circuit breaking.",
-  "version": "4.1.0",
+  "version": "4.2.0",
```

Comment:
Remove; there is a release workflow that deals with versioning.  If you have change log guidance add it to the commit.

See: https://www.conventionalcommits.org/en/v1.0.0/